specification/healthdataaiservices/HealthDataAIServices.DeidServices/routes.tsp

```diff
@@ -31,23 +33,45 @@ alias DeidentifyOperations = ResourceOperations<
 #suppress "@azure-tools/typespec-azure-core/use-standard-names" "Existing name"
 @summary("Create a de-identification job.")
 @pollingOperation(getJob)
-op createJob is DeidentifyOperations.LongRunningResourceCreateOrReplace<DeidentificationJob>;
+op deidentifyDocuments is DeidentifyOperations.LongRunningResourceCreateOrReplace<DeidentificationJob>;
 
 @summary("Get a de-identification job.")
 op getJob is DeidentifyOperations.ResourceRead<DeidentificationJob>;
 
 @summary("List de-identification jobs.")
+@access(Access.internal, "csharp,python")
 op listJobs is DeidentifyOperations.ResourceList<
   DeidentificationJob,
   ListQueryParametersTrait<PaginationByTokenQueryParameters>
 >;
 
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations" ""
 @summary("List processed documents within a job.")
-op listJobDocuments is DeidentifyOperations.ResourceList<
-  DocumentDetails,
-  ListQueryParametersTrait<PaginationByTokenQueryParameters>
+@route("/jobs/{jobName}/documents")
+@access(Access.internal, "csharp,python")
+op listJobDocuments is Azure.Core.Foundations.Operation<
+  {
+    @doc("The name of a job.")
+    @maxLength(36)
+    @minLength(3) // Must be 3 to match regex
+    @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_]+[a-zA-Z0-9]$")
+    jobName: string;
+
+    ...PaginationByTokenQueryParameters;
+  },
+  Page<DeidentificationDocumentDetails>,
+  Azure.Core.Foundations.ErrorResponse
 >;
```

Comment:
Why do we replace a standard operation by a non-standard one?

Reply:
I've changed it back to the standard `ResourceList` operation - thanks for flagging